manager/scheduler/scheduler.go

```diff
@@ -1,14 +1,13 @@
 package scheduler
 
 import (
-	"time"
-
 	"github.com/docker/swarmkit/api"
 	"github.com/docker/swarmkit/log"
 	"github.com/docker/swarmkit/manager/state"
 	"github.com/docker/swarmkit/manager/state/store"
 	"github.com/docker/swarmkit/protobuf/ptypes"
 	"golang.org/x/net/context"
+	"time"
```

Comment:
Move `"time"` back to the top of the `import` statement, and separate it from the other imports with a blank line.